test/vcf/metaentry_test.cpp

```diff
@@ -38,12 +38,12 @@ namespace ebi
             
             SECTION ("It should work with any ID and source")
             {
-                CHECK_NOTHROW( (vcf::MetaEntry { 1, "reference" }) );
+                CHECK_NOTHROW( (vcf::MetaEntry { 1, "reference", std::make_shared<vcf::Source>(source) }) );
```

Comment:
I'd say the constructor call will be easier if you replace the type of the source to `shared_ptr`:
```
std::shared_ptr<Source> source{
        std::make_shared<vcf::Source>(
                "Example VCF source",
                vcf::InputFormat::VCF_FILE_VCF | vcf::InputFormat::VCF_FILE_BGZIP,
                vcf::Version::v41,
                vcf::Ploidy{2},
                {},
                { "Sample1", "Sample2", "Sample3" })};
```
and then I would do a find&replace of `std::make_shared<vcf::Source>(source)` to `source`. the same change applies to record_test.cpp.

also, the indentation of the `SECTION`s in the 2 first `TEST_CASE`s is wrong.